src/lang/en/validation.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
 return [
-    'profane' => 'The :attribute contains vulgar content',
+    'profane' => 'Di :attribute di na le matlhapa',
```

Comment:
Please fix this file. The `en/validation.php` must contains the text in english. I think you confuse the files